lighthouse-core/lib/i18n/i18n.js

```diff
@@ -151,6 +151,24 @@ const formats = {
   },
 };
 
+/**
+ * Function to augment a given locale with additional messages.
+ * @param {LH.Locale} locale
+ * @param {LH.LocaleMessages} messages
+ * @param {string=} dir Optional absolute path to be appended to message keys.
+ */
+function augmentLocale(locale, messages, dir) {
+  if (dir) {
+    for (const [key, message] of Object.entries(messages)) {
+      delete messages[key];
+      const [filename, varName] = key.split(' | ');
+      const relativeDir = path.relative(LH_ROOT, dir).replace(/\\/g, '/');
+      messages[`${path.join(relativeDir, filename)} | ${varName}`] = message;
```

Comment:
share these few lines to build `i18nId` with `getIcuMessageFn`? :) 

I could see us forgetting to update this if that format ever changed

Reply:
Done.